docker/Dockerfile.cpu

```diff
@@ -2,8 +2,10 @@ FROM ubuntu:latest
 
 COPY install.sh /
 
-RUN /install.sh && pip uninstall --yes mxnet && pip install mxnet-mkl
+RUN /install.sh && pip uninstall --yes mxnet && pip install mxnet-mkl && mkdir /mxnet_model_server
 
-ADD mms_docker_cpu mms_docker_cpu
+COPY wsgi.py setup_mms.py mxnet-model-server.sh /mxnet_model_server/
+
+ENV PATH="/mxnet_model_server:${PATH}" GPU_IMAGE=0
```

Comment:
GPU_IMAGE -> MXNET_MODEL_SERVER_GPU_IMAGE

Reply:
Sure. I will change this.